resources/assets/templates/vote/create-vote-header.hbs

```diff
@@ -0,0 +1,80 @@
+<div class="form-group">
+    <label for="usr">Question:</label>
+    <input type="text" class="form-control" id="question-title" value="{{model.title}}" {{#ifCond meta.editable '==' false}}disabled{{/ifCond}}>
+    <div class="js-errors js-error-title"></div>
+</div>
+
+<div class="form-group">
+    <label>Description:</label><br>
+    <textarea rows="5" class ="form-control description" id="question-description" {{#ifCond meta.editable '==' false}}disabled{{/ifCond}}>{{model.description}}</textarea>
+    <a class="openMarkdownHelp">Open Markdown Help </a>
+    <div class="js-errors js-error-description"></div>
+</div>
+
+<div class="form-group">
+    <label>Sluggable Url:</label>
+    <input type="text" class="form-control" id="question-slug" value="{{model.slug}}">
+    <div class="js-errors js-error-slug"></div>
+</div>
+
+<div class="form-group">
+    <label for="usr">Tags(Split them using spaces, no #):</label>
+    <input type="text" id="tags" class="form-control" value="{{meta.tags}}" {{#ifCond meta.editable '==' false}}disabled{{/ifCond}}>
```

Comment:
You can use basic handlebars operator
{{#unless meta.editable}}{{/unless}}
instead of 
{{#ifCond meta.editable '==' false}}